bin/file_extension.py

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env python3
+
+import sys
+
+from reusable import json_dump, package_validation_dict
+
+
+def file_extension_test(file_path: str):
+    """
+    1. Reads file extension from provided path
+    2. Checks for valid file extension (.txt is used for now, future version will use ome.tiff and ome.tif)
+    3. Returns result of File extension test ("pass" or "fail")
+
+    Args:
+        file_path (str): Path to the downloaded file
+
+    Returns:
+        str: Result of File extension test
+    """
+    ext = file_path.split(".", 1)[-1]
+    # Do we want the extension check to be hard-coded like this or check against the file type data provided in the CSV?
```

Comment:
Should this check be done against the `file_type` data that is provided in the input CSV file? It is hard-coded to check for "ome.tiff" or "ome.tif" at the moment.

Reply:
My 2 cents is that we would need to use the `file_type` data to figure out which validation module to run.